docs/Getting Started/Fedora/Fedora Root On ZFS.rst

```diff
@@ -629,9 +629,9 @@ Step 8: Last Minute Fixes
 
      sed -i 's/#timeout.*/timeout 10/' /boot/loader/loader.conf
 
-#. Ensure that zfs-mount.service is enabled::
+#. Ensure that zfs-target is enabled::
 
-     systemctl enable zfs-mount.service
+     systemctl enable zfs.target
```

Comment:
I've recently encountered a Fedora system that failed to mount the zfs pools during boot because the zfs kernel module was not available in time. I found that adding `rd.driver.pre=zfs` to the list of kernel parameters resolved the problem. I suspect that may be what is needed here instead of `systemctl enable zfs.target`. The zfs target should be enabled by default.

Just FYI.

Reply:
the zfs.target isnt to solve thats, its to solve the edge case in which the zfs pools and partitions arent loaded for some reason

Reply:
If there is even a chance that the system could fail to boot due to sub-filesystems failing to mount, I'd be tempted to minimize the layout down to just / and /home (if /home fails to mount, it will not prevent the system from booting and it is easy to mount after booting using the root account). It looks like your guide is pretty close to doing that already. Also, I _think_ that is pretty close to how Fedora 33 is going to layout their default btrfs file system: https://fedoramagazine.org/btrfs-coming-to-fedora-33/#comment-502491

I see that your current version has /var/log and /var/spool as separate file systems. If they were to fail to mount, the system should still boot, but it might be good to note that if they fail once, they will continue to fail on subsequent boots/attempts if any files are left under those mountpoints. So it might be helpful to inform the user that they will need to delete auto-generated files under those directories to recover from the error. Or better yet, IMHO, just keep them on the / file system by default and let the user know that if they don't want their logs rolled back, then they will need to make /var/log a separate file system.

I still haven't tried it out, but your guide is looking prettying good so far. Thanks for doing the research on this.

Reply:
Your welcome